extensions/amp-form/0.1/test-e2e/test-amp-form.js

```diff
@@ -0,0 +1,44 @@
+/**
+ * Copyright 2019 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+describes.endtoend('amp-form SSR templates', {
```

Comment:
This test file scheme is confusing.
- `test-amp-form.js` tests the SSR case.
- `test-ssr.js` tests the non-SSR case.

Let's merge both into `test-ssr.js` but have two `describes.endtoend` blocks in one file.